adapters/statsd-report-status-code.js

```diff
@@ -15,11 +15,13 @@ function handleReportingRequest(request, requestOptions, handleResponse) {
     var self = this;
     self.requestHandler.request(request, requestOptions, onResponse);
     function onResponse(error, response) {
+        var resource = requestOptions.resource;
+        var statsEmitter = self.options.statsEmitter;
         if (error) {
+            // report 500 for client errors, e.g. ETIMEDOUT, ESOCKETTIMEDOUT
+            statsEmitter.emit('statusCode', resource, 500);
```

Comment:
In fact; for http errors lets just report `err.code` and for airlock lets just report `'airlock'`